scripts/clean-shell-files.ps1

```diff
@@ -0,0 +1,7 @@
+$directories = Get-ChildItem -Path $env:MainDirectory -Directory -Exclude @("core", "scripts", ".github", ".git")
```

Comment:
I had core originally included in there because of a submodule on the repo, if you don't mind cleaning up core from the Go script please?